clar2wasm/src/words/conditionals.rs

```diff
@@ -213,6 +213,118 @@ impl Word for Filter {
     }
 }
 
+#[derive(Debug)]
+pub struct And;
+
+impl Word for And {
+    fn name(&self) -> ClarityName {
+        "and".into()
+    }
+
+    fn traverse(
+        &self,
+        generator: &mut WasmGenerator,
+        builder: &mut walrus::InstrSeqBuilder,
+        _expr: &SymbolicExpression,
+        args: &[SymbolicExpression],
+    ) -> Result<(), GeneratorError> {
+        let n_branches = args.len();
+
+        let mut branches = vec![];
+
+        let noop = builder
+            .dangling_instr_seq(InstrSeqType::new(
+                &mut generator.module.types,
+                &[],
+                &[ValType::I32],
+            ))
+            // for now, the noop branch just adds a false to break out of the next iteration
+            .i32_const(0)
+            .id();
+
+        for i in 0..n_branches {
+            let branch_expr = args.get_expr(i)?;
+
+            let mut branch = builder.dangling_instr_seq(InstrSeqType::new(
+                &mut generator.module.types,
+                &[],
+                &[ValType::I32],
+            ));
+
+            generator.traverse_expr(&mut branch, branch_expr)?;
+
+            branches.push(branch.id());
+        }
+
+        builder.i32_const(1);
+
+        for branch in branches {
+            builder.instr(ir::IfElse {
+                consequent: branch,
+                alternative: noop,
+            });
+        }
```

Comment:
Since this implementation is pretty much similar to both `and` and `or` functions, can we have this logic in a separate function, that will receive the noop branch conditional as an argument?